### ISSUE TITLE
My device doesn't switch unless the other one is disabled

### ISSUE THREAD

**Issue by MadPanCake** [xenolightning/AudioSwitcher_v1#2225]
Please ensure you include/confirm the following information before submitting an issue.

- [x] I have listed the Audio Switcher version
- [x] I have listed my operating system, and version
- [x] Include any details about the audio devices on the system
- [ ] If you are using Nvidia or AMD for sound, include the driver versions
- [x] This is not an issue about setting more than one device as default from a hotkey, quick switch or other relating switching activity.

And do remember this application is developed and distributed **FOR FREE**, you should have no expectation that an issue will be fixed in a reasonable time-frame.

-----

Hello,
Audioswitcher doesn't work for me, as in my system, I have to disable the other device, in order to make one work. I don't know why is that.
So, If I want to switch between my headphones and speakers, I just disable speakers, and then headphones work, and vice versa.

I wonder how can I fix that, so Audioswitcher can work for me, and free me from the hassle of switching haha.

My headphones are connected With Focusrite Scarlett 2i2 sound interface, and my speakers are connected directly into my PC through the line output (the green port at the back of the PC).
I'm using the latest Audioswitcher version.

Again, switching devices with Audioswitcher does nothing, the same device keeps playing. The switch ONLY occurs, when one device is disabled, so there is only one device enabled at a time, after that I can enable any number of devices, and selecting one of them as default (communication) device doesn't change nothing at all.

**Comment by xenolightning** [comment]
@MadPanCake - does switching via the Windows Control Panel Sound menu? It really shouldn't be any different than the Control Panel way of switching, so good to confirm that works first.

Also useful to know what version of Windows you are using

**Comment by MadPanCake** [comment]
Hello @xenolightning, thanks for answering my issue.

I realized yesterday, that I don't actually have to disable one of the devices for them to switch. In reality, the favorite device, and favorite communications device does switch when using the program, and when doing that manually.
It's just that some of the programs switch immediately, and some require disabling the other device to switch to the only one active device. And that's where my problem lies.

-----

Update: Checked it now, when my audio interface(headphones) is the selected device first and it plays music on youtube, Audioswitcher can actually switch my audio device ONCE, to speakers. But it can't switch it back to headphones (it switched default/favorite device thing, but it doesnt change sound source)
Same thing goes with Apex Legends, and probably other programs.
**So to sum it up, the issue is it actually is capable of only switching from Audio Interface, to my Speakers, but not vice versa.**

----

Ideally, Audioswitcher could get a feature that could disable the other device, because I know I'm not the only person that has this problem.

Answering your question: Yes, switching works via the windows control panel sound menu, although, Google Chrome never suffices from just selecting the favorite device in order to switch the sound source, I always have to additionally disable the other device in order for it to work.

Answering your other question: My Windows version is Windows 10 Pro, 21H2, 19044.1645
![image](https://user-images.githubusercontent.com/34662922/168467652-59742619-8332-405d-9535-c396fc71d48d.png)
